### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ananya88**
Fix some README.md typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ project. The `openr` directory contains the source for the project.
 ### Dependencies
 
 OpenR requires these dependencies for
-your system and follow the traditional cmake build steps below.
+your system and  follows the traditional cmake build steps below.
```
Comment (cooperlees) [inline README.md:71]:
```suggestion
your system and follows the traditional cmake build steps below.
```

File: README.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ sudo python setup.py install
 
 ### Docker Building / Usage
 
-OpenR not has a `Dockerfile`. It uses `gendeps.py` to build all dependencies + OpenR.
+OpenR does not have a `Dockerfile`. It uses `gendeps.py` to build all dependencies + OpenR.
```
Comment (cooperlees) [inline README.md:141]:
```suggestion
OpenR does now has a `Dockerfile`. It uses `gendeps.py` to build all dependencies + OpenR.
```

**Comment by Ananya88** [comment]
@cooperlees Thanks for your response , it would be of great help if you can tell how can I resolve build boost failed case of linux/build .

File: README.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ sudo python setup.py install
 
 ### Docker Building / Usage
 
-OpenR not has a `Dockerfile`. It uses `gendeps.py` to build all dependencies + OpenR.
+OpenR does now has a `Dockerfile`. It uses `gendeps.py` to build all dependencies + OpenR.
```
Comment (cooperlees) [inline README.md:141]:
```suggestion
OpenR now has a `Dockerfile`. It uses `gendeps.py` to build all dependencies + OpenR.
```
Is “does now” correct English?

**Pull request closed without merging**